libraries/Microsoft.BotKit.Adapters.Slack/ActivityResourceResponse.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.BotKit.Adapters.Slack
```

Comment:
The Namespace for these should be our usual namespace:
Microsoft.BotBuilder.Adapters.Slack

Reply:
We renamed the projects and namespaces. Thanks!